npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater.rb

```diff
@@ -25,6 +25,17 @@ def raven_context
         end
       end
 
+      class GroupedSubDependencyUpToDate < StandardError
+        def initialize(message:, error_context:)
+          super(message)
+          @error_context = error_context
+        end
+
+        def raven_context
+          { extra: @error_context }
+        end
+      end
```

Comment:
I think my concern with adding this is that it makes npm specifically semi-aware of groups but in a way where it is kind of guessing the problem is due to a grouped update being performed 🤔 

This could lead to confusion where due to a bug elsewhere this error starts appearing in non-grouped updates?